@kiva/kv-components/src/vue/stories/KvLendCta.stories.js

```diff
@@ -320,7 +318,6 @@ export const WithPresetOptionsHugeAmount = story({
 	},
 	unreservedAmount: '12850.00',
 	kvTrackFunction,
-	enableHugeAmount: true,
```

Comment:
We can remove these stories dedicated to the "huge amount".

Reply:
I did it, but reverted then as are the cases we have to visualize that feature, wdyt?

Reply:
Isn't the huge lending amount just a default now in the other stories?

Reply:
Ah that makes sense, thanks, we can keep the stories then.